readthedocs/core/utils/objects.py

```diff
@@ -0,0 +1,28 @@
+_DEFAULT = object()
+
+
+def get_dotted_attribute(obj, attribute, default=_DEFAULT):
```

Comment:
Seems like this is something already in the stdlib, or I imagine must exist in some reusable form? 

https://stackoverflow.com/a/59480744

Reply:
That's for accessing attributes from a dictionary. What we are using this for, is to dynamically get nested fields (e.g, similar to when using getattr).